### PR TITLE
EIA: Fix date parsing

### PR DESCRIPTION
**Pull request by pradh**
Parse full date, which appears in PET dataset, correctly.

### CODE REVIEW

File: scripts/us_eia/opendata/process/common.py [hunk 1]
```diff
@@ -38,6 +38,11 @@
 def _parse_date(d):
     """Given a date from EIA JSON convert to DC compatible date."""
 
+    try:
+      _ = int(d)
```
Comment (beets) [inline scripts/us_eia/opendata/process/common.py:42]: could d.isnumeric() work here?

Comment (beets) [reply scripts/us_eia/opendata/process/common.py:42]: out of curiosity, what kinds of non-numeric values did you come across? should we add some to the test?

**Review by beets — APPROVED** [review]
thx!

File: scripts/us_eia/opendata/process/common.py (hunk 1 above)
Comment (pradh) [reply scripts/us_eia/opendata/process/common.py]: I added it as an extra check, but there is actually no non-numeric value so far (ELEC, NG, PET).

Yes, isnumeric() is nicer!

**Review by pradh — COMMENTED** [review]
Thanks!

File: scripts/us_eia/opendata/process/common.py (hunk 1 above)
Comment (pradh) [reply scripts/us_eia/opendata/process/common.py]: My comment about non-numeric is not true. There is 2019Q1 pattern. I'll add a test in upcoming PR.

**Pull request merged**